pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -641,8 +650,15 @@ func validateNodeGroupBase(np NodePool, path string, controlPlaneOnOutposts bool
 		}
 	}
 
+	instanceType := SelectInstanceType(np)
+
+	if ng.AMIFamily == NodeImageFamilyAmazonLinux2023 && instanceutils.IsNvidiaInstanceType(instanceType) {
+		return ErrUnsupportedInstanceTypes("GPU", NodeImageFamilyAmazonLinux2023,
+			fmt.Sprintf("EKS accelerated AMIs based on %s will be available at a later date", NodeImageFamilyAmazonLinux2023))
```

Comment:
Is there a communication date on when this will be available? happy to ask around internally